client-spark/common/src/main/java/org/apache/spark/shuffle/writer/WriteBufferManager.java

```diff
@@ -316,6 +321,9 @@ public synchronized List<ShuffleBlockInfo> clear() {
             + dataSize
             + "], memoryUsed["
             + memoryUsed
+            + "],"
+            + "block number["
+            + result.size()
```

Comment:
I think `number of blocks` is easier to understand:
```suggestion
            + "], number of blocks["
            + result.size()
```

or simply `blocks`:
```suggestion
            + "], blocks["
            + result.size()
```

The term `block number` is too similar to `sequence number` in blockId.